cms/src/main/java/uk/nhs/hee/web/validation/validator/AbstractMandatoryValidator.java

```diff
@@ -26,7 +26,7 @@ public AbstractMandatoryValidator(List<String> fields) {
     public Optional<Violation> validate(ValidationContext context, final Node node) {
         try {
             for (String fieldName : combinationFields) {
-                if (node.getProperty(fieldName).getValues().length > 0) {
+                if (node.hasProperty(fieldName) && node.getProperty(fieldName).getValues().length > 0) {
```

Comment:
Apparently, this condition doesn't handle the scenario wherein both `Professions` and `Healthcare topics` fields empty when the `News article` document is being created for the first time. We may need to rewrite the condition, thanks